tools/exportNotebook

```diff
@@ -361,18 +362,18 @@ else
     echo "Left temporary dir '${tmpfolder}'"
 fi
 
-if [ -n $ASK_RENAME ]; then
+if [ -n "$ASK_RENAME" ]; then
     xdg-open "${filename}.pdf"
     [ -n "$OUTPUT_UUID" ] && echo "Last modified: ${DOCUMENTS_DATES[$OUTPUT_UUID]}"
     echo
-    read -rp "New name w/o extension, or press Enter to delete files: " NEWFILENAME
+    read -rp "New name w/o extension, '!rm' to delete files, Enter to skip: " NEWFILENAME
```

Comment:
May I suggest a consistency improvement with usual CLI tools, to show the value used in case no entry is made?

```suggestion
    read -rp "New name w/o extension, '!rm' to delete files [${filename}]: " NEWFILENAME
```

Reply:
fixed, see https://github.com/reHackable/maxio/pull/29/commits/3cb7b765c2107449b434b8af1053ab51583106d0